html/modules/custom/docstore/src/ResourceTrait.php

```diff
@@ -549,8 +549,19 @@ public function prepareEntityEntityReferenceField(EntityReferenceFieldItemList $
       // @todo It would much easier if we were to use "name" for terms which is
       // the field name for the term label rather than "label" or if we were to
       // use "label" for all the entity types.
-      if ($entity->getEntityTypeId() === 'taxonony_term') {
-        $label_key = 'label';
+      if ($entity->getEntityTypeId() === 'taxonomy_term') {
+        // @todo This was included when there was a typo in 'taxonomy_term',
+        // but it causes tests to fail, so specifying 'name' for now.
```

Comment:
`label` is the property name used when creating or retrieving terms so for consistency it should be 
"label" as well here and the tests modified. That may have an impact on the current AR (not anymore normally after https://github.com/UN-OCHA/assessmentregistry8-site/pull/233 is merged).